README.md

```diff
@@ -37,9 +49,9 @@ Submit a pull request to add a link to your repository here.
 
 - ✓ [Canlista](https://k7gat-daaaa-aaaae-qaahq-cai.ic0.app/)
 
-- [Motoko Top Up Canister](https://github.com/ORIGYN-SA/motoko_top_up_canister) - A Motoko service for topping up canisters
+- [Motoko Top Up Canister](https://github.com/ORIGYN-SA/motoko_top_up_canister) - A Motoko service for topping up canisters.
```

Comment:
The [awesome-lint](https://github.com/sindresorhus/awesome-lint) tool is very particular about punctuation and spelling GitHub as Github, instead of Github 😅